src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -88,15 +88,15 @@ public static Phone parsePhone(String phone) throws ParseException {
     }
 
     /**
-     * Parses a {@code String Date_of_birth} into a {@code DateOfBirth}.
+     * Parses a {@code String dob} into a {@code DateOfBirth}.
      * Leading and trailing whitespaces will be trimmed.
      *
      * @throws ParseException if the given {@code Date_of_birth} is invalid.
```

Comment:
I think you missed changing this to `dob`
```suggestion
     * @throws ParseException if the given {@code dob} is invalid.
```